docs/next_steps_nitrokey.md

```diff
@@ -0,0 +1,38 @@
+# Propose and document new API design based on CTAPHID
+
+With CTAPHID used instead of CoAP, human-readable names of endpoints should be
+changed to numeric IDs. CoAP status reporting also has to be modified to account
+for different interface.
+
+# Implement functions specific to Fobnail in libnitrokey
+
+This is temporary solution to allow testing until full Fobnail support is added
+to libnitrokey.
```

Comment:
Testing aspect is always quite important in these applications. We could at least mention that the library will be covered with unit test to ensure the proper implementation of the documented API.

Reply:
Added.